omod/src/main/resources/apps/dataintegrity_app.json

```diff
@@ -12,5 +12,28 @@
         "order": 300
       }
     ]
+  },
+  {
+    "id": "coreapps.dataIntegrityViolations",
+    "instanceOf": "coreapps.template.dashboardWidget",
+    "description": "dataintegrity.desc.label",
+    "order": 10,
+    "config": {
+      "widget": "dataintegrityviolations",
+      "icon": "icon-question-sign",
+      "label": "Data Quality Violations",
+      "maxResults": "8"
+    },
+    "extensions": [
+      {
+        "id": "org.openmrs.module.coreapps.mostRecentVitals.clinicianDashboardFirstColumn",
+        "appId": "ugandaemr.dataIntegrityViolations",
```

Comment:
@ssmusoke , I've made the change you're suggested but now I'm getting new error. Here is full log: https://pastebin.com/V1xXyy1L

After getting the error I further changed the `appId` to `coreapps.dataIntegrityViolations` then I got no error but I wasn't able to see the content of `data quality widget`.